doc/tools.rst

```diff
@@ -0,0 +1,30 @@
+.. _tools:
+
+:octicon:`terminal` Tools
+=========================
+
+The python-toolbox ships with a set of command line tools, whose entry point always is the command :code:`tbx`.
+The commands are structured in a *tree* manner, and help is provided along with the command(s) no matter the nesting.
+
+How to get Help
+---------------
+
+.. code-block:: shell
+
+     $ tbx --help
+
+.. code-block:: shell
+
+     $ tbx command --help
+
+.. code-block:: shell
+
+     $ tbx command subcommand --help
+
+.. code-block:: shell
+
+     $ tbx command subcommand subsubcommand --help
```

Comment:
It may sound redundant to some of us, but I really like this kind of verbosity at least in one place.
Thank you @Nicoretti !